### PR TITLE
Fixes for v1.4 (Issues 196, 165)

### LINKED ISSUE #165: [PE-D][Tester A] Unable to see the full names of people with long names
As described, a very small feature flaw as there is no way to see the full name.

Input:
Add a very long name as string in add command

-------------
Labels: `type.FeatureFlaw` `severity.VeryLow`
original: JerryO3/ped#6

### LINKED ISSUE #196: [PE-D][Tester E] Find + delete command does not update meetings list as intended
<i><video controls><source src="https://raw.githubusercontent.com/yyccbb/ped/main/files/2a460890-9928-4200-8f70-583fe6a3c5a0.mp4" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/yyccbb/ped/main/files/2a460890-9928-4200-8f70-583fe6a3c5a0.mp4</i>

Can see that delete command actually deletes the corresponding index in the meeting list instead of searching through the meetings list and find the entry that matches the client to be deleted.

-------------
Labels: `type.FunctionalityBug` `severity.High`
original: yyccbb/ped#14

### PR DESCRIPTION
**Pull request by rahhulleee**
Close #196
Close #165

### CODE REVIEW

**Review by youdonnnn — APPROVED** [review]
lgtm!

File: src/main/java/seedu/address/model/person/Name.java [hunk 1]
```diff
@@ -10,7 +10,8 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain alphanumeric characters and spaces, and it should not be blank,\n"
+            + "and it should be no more than 20 characters";
```
Comment (irwynliong) [inline src/main/java/seedu/address/model/person/Name.java:14]: no more than 40 characters

**Review by irwynliong — CHANGES_REQUESTED** [review]
please re commit

File: src/main/java/seedu/address/model/person/Name.java (hunk 1 above)
Comment (jimseah-0116) [reply src/main/java/seedu/address/model/person/Name.java:14]: I'll fix it, no worries

**Review by irwynliong — APPROVED** [review]

**Pull request merged**